docs/src/main/sphinx/admin/properties-task.rst

```diff
@@ -111,6 +111,32 @@ resource utilization, but uses additional memory.
 The minimum number of drivers guaranteed to run concurrently for a single task given
 the task has remaining splits to process.
 
+``task.scale-writers``
+^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Scale the number of concurrent writers per task based on throughput. The maximum
+scale it can reach is equal to the maximum of ``task.writer-count`` and
+``task.min-scaled-writer-count``. Also, note that the default value of
+``task.min-scaled-writer-count`` is 8, which means it'll scale up to 8 concurrent
+writers. This can be specified on a per-query basis using the ``task_scale_writers``
+session property.
+
+``task.min-scaled-writer-count``
```

Comment:
Move this to "Writer scaling properties"

Reply:
agreed .. although the namespace is a bit messed up since its using task for both..

Reply:
I've added a link